web/core/src/jsMain/kotlin/androidx/compose/web/attributes/TextAreaAttrsBuilder.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package androidx.compose.web.attributes
+
+import org.jetbrains.compose.web.attributes.*
+import org.jetbrains.compose.web.events.WrappedTextInputEvent
+import org.w3c.dom.HTMLTextAreaElement
+
+class TextAreaAttrsBuilder : AttrsBuilder<HTMLTextAreaElement>() {
+
+    fun onInput(options: Options = Options.DEFAULT, f: (SyntheticInputEvent<String, HTMLTextAreaElement>) -> Unit) {
```

Comment:
let's have some more obvious param names or discuss why "f" would be the best choice here )

Reply:
fixed